### DISCUSSION TITLE
How to edit "Confirm reauthentication" email template?

### DISCUSSION THREAD

**Discussion by elliotwaite** [supabase/supabase#15639] [Questions] [answered]
Is there a way to edit the "Confirm reauthentication" email template that gets sent after calling `supabase.auth.reauthenticate()`? I don't see a customizable template for it listed in the **Authentication > Email Templates** section. The code for the default message body can be found [here](https://github.com/supabase/gotrue/blob/77afd2834201e50672502a48bdc365b4ba7a095b/internal/mailer/template.go#L67), and it looks like this:

Subject heading: Confirm reauthentication

Message body:
### Confirm reauthentication
Enter the code: 458912

**Comment by GaryAustin1** [comment, marked as answer]
It does not appear to be implemented yet.  That call was just added to the docs  3 weeks ago.  https://github.com/supabase/gotrue/issues/1015
You should generate an issue in that repository.

**Reply by elliotwaite** [reply]
@GaryAustin1, thanks for the info and the suggestion. I opened an issue in that repo: https://github.com/supabase/gotrue/issues/1173
